.env.sample

```
@@ -1,2 +1,6 @@
 PORT=5000
-USE_TLS=false
\ No newline at end of file
+USE_TLS=false
+
+# Supabase Logging (optional)
+# SUPABASE_URL=https://alclkpvfnmwsfuzlstfo.supabase.co
+# SUPABASE_SERVICE_KEY=your-service-role-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)